_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -5109,9 +5109,9 @@ mfxU64 make_back_color_argb(mfxU16 bit_depth, mfxU16 R, mfxU16 G, mfxU16 B)
     mfxU64 const max_val = (1 << bit_depth) - 1;
 
     return ((mfxU64) max_val << 48) |
-           ((mfxU64) VPP_RANGE_CLIP(R, 0, max_val) << 32) |
-           ((mfxU64) VPP_RANGE_CLIP(G, 0, max_val) << 16) |
-           ((mfxU64) VPP_RANGE_CLIP(B, 0, max_val) << 0);
+           ((mfxU64) mfx::clamp<mfxU64>(R, 0, max_val) << 32) |
+           ((mfxU64) mfx::clamp<mfxU64>(G, 0, max_val) << 16) |
+           ((mfxU64) mfx::clamp<mfxU64>(B, 0, max_val) << 0);
 }
```

Comment:
No need to cast to mfxU64 in return computations.

Reply:
right